geode-core/src/test/java/org/apache/geode/management/internal/cli/domain/IndexDetailsIntegrationTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.cli.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.geode.cache.Cache;
+import org.apache.geode.cache.Region;
+import org.apache.geode.cache.RegionShortcut;
+import org.apache.geode.cache.query.Index;
+import org.apache.geode.distributed.DistributedMember;
+import org.apache.geode.test.junit.categories.IntegrationTest;
+import org.apache.geode.test.junit.rules.ServerStarterRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+@Category(IntegrationTest.class)
+public class IndexDetailsIntegrationTest {
+
+  private static final String REGION_1 = "REGION1";
+  private static final String INDEX_REGION_NAME = "/REGION1";
+  private static final String INDEX_1 = "INDEX1";
+
+  private Region region;
+
+  @Rule
+  public ServerStarterRule serverRule =
+      new ServerStarterRule().withRegion(RegionShortcut.REPLICATE, REGION_1).withAutoStart();
```

Comment:
if you have withRegion, then you don't need autoStart(), the before() in the rule sees you need a region, so it will start the server anyway first to create the region.